modules.d/90dmsquash-live/module-setup.sh

```diff
@@ -22,7 +22,7 @@ installkernel() {
 
 # called by dracut
 install() {
-    inst_multiple umount dmsetup blkid dd losetup blockdev find rmdir
+    inst_multiple umount dmsetup blkid dd losetup blockdev find rmdir grep sed
```

Comment:
sed is already installed by the base module, which is a dependency to this dmsquash-live module, so no use to repeat it here.

Reply:
Ah, ok, I'll drop that.